File: tdd-workflow-example-php/src/Greeting/Greeter.php
<?php

namespace Greeting;

final class Greeter
{

    public function greet(string $name) : string
    {
        if ('' === $name) {
            return 'Hi!';
        }

        return "Hello, $name!";
    }

    public function loudGreet(string $name) : string
    {
        return strtoupper($this->greet($name));
    }
}

File: tdd-workflow-example-java/src/main/java/greeting/Greeter.java
package greeting;

public class Greeter {
    public String greet(String name) {
        if ("".equals(name)) {
            return "Hi!";
        }

        return "Hello, " + name + "!";
    }

    public String loudGreet(String s) {
        return greet(s).toUpperCase();
    }
}
